vcf/filter.go

```diff
@@ -40,6 +40,9 @@ func FilterQual(v *Vcf, minQual float64) bool {
 //FilterAlt returns true if the Alt field of a Vcf entry matches a desired input Alt field, false otherwise. Order sensitive.
 func FilterAlt(v *Vcf, alt []string) bool {
 	if len(alt) > 0 && CompareAlt(v.Alt, alt) != 0 {
+		if alt[0] == "" {
```

Comment:
I am not sure I understand this part.  It seems like if the first entry of the desired alt field is empty this is treated as a special value that overrides other logic to return true, but I am not sure why.  it also seems like passing in an alt value that is empty will cause this to always match

Reply:
So we have the case where the Alt option in the cmd is left empty, which currently is signified by "". Alt in this case is the user inputed slice of strings corresponding to the desired state of Alt in the output file, if this is empty, then we want this to pass no matter what.